### PR TITLE
Split AWS release into Asset Store & Belly

### PR DESCRIPTION
**Pull request by manelizzard**

### CODE REVIEW

File: .github/workflows/homa-belly-release.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-name: AWS release
+name: Homa Belly release
```
Comment (homapf) [inline .github/workflows/homa-belly-release.yml:1]: Do we want Homa Belly things to be public ?

Comment (manelizzard) [reply .github/workflows/homa-belly-release.yml:1]: Oh that's true, this repo should be public. Then I would completely remove this workflow and have it inside Homa Belly related projects. WDYT?

**Comment by ClementCMobile** [comment]
We don't want Homa Belly stuff to be public, but we push Asset Store specific stuff here? I am not following
 If we want to be able to push with 2 different AWS "destination", why not just keep the "AWS" workflow, and add the destination as parameter?

**Comment by homapf** [comment]
Agree with @ClementCMobile here, I think we just want a generic AWS workflow that can be reused 👍

**Review by ClementCMobile — APPROVED** [review]

**Pull request merged**